common/buildcraft/transport/BCTransportRegistries.java

```diff
@@ -67,12 +67,13 @@ public static void init() {
         PipeApi.stripeRegistry.addHandler(StripesHandlerPlaceBlock.INSTANCE, EnumHandlerPriority.LOW);
         PipeApi.stripeRegistry.addHandler(StripesHandlerUse.INSTANCE, EnumHandlerPriority.LOW);
 
-        StripesHandlerDispenser.ITEM_CLASSES.add(ItemBucket.class);
-        StripesHandlerDispenser.ITEM_CLASSES.add(ItemMinecart.class);
+        // For testing
```

Comment:
The ITEM_CLASSES list is only used in a commented out section which has a for testing remark